### ISSUE TITLE
TextureFileData.db2 build 7.3.5 25XXX

### ISSUE THREAD

**Issue by Leschen** [WowDevTools/WDBXEditor#51]
Hi again, sorry but your solution does not seem to work even after getting the size limit of the ID column
being an int22. I don't understand cause base texture work but not the added ones. My brain hurts x)

**Comment by barncastle** [comment]
It sounds like the issue is with your custom data opposed to the file. If WDBX had saved this incorrectly the whole file wouldn't work not just new ones.

Also, are you sure the new file is being loaded and not the original one?

**Comment by Leschen** [comment]
I mean the new files all work when I have to use the root ID instead of the texturefiledata one.
When i use a newly added row that is not an original game line  it just show either blank texture or shit texture. No problem with original game ones. Meaning the ID is not the problem. I respected the in22 limit. And I did a corresponding ID by auto-increment of the last official one to be sure. Nothing works. Maybe there's a non-documented flags or I don't know but it does not work at all. I'm not alone on this problem. CSV import ok, brute hand OK, SQL ok but in game nope.

This bug was worse in 7.2. When we added a line to it with wdbx in 7.2 it would just literally fuck up even officail game's texture by fucking up the texture layout etc.

I can't give more details. It just does not work :\
